src/Session.cpp

```diff
@@ -450,26 +450,6 @@ bool CSession::InitializeDRM(bool addDefaultKID /* = false */)
       AP4_DataBuffer init_data;
```

Comment:
since the variable `currentSessionTypes` is not used anymore must be removed, on line 445 (gh dont allow to comment to unchanged code so i comment here)